.prettierignore

```diff
@@ -0,0 +1,6 @@
+dist/
```

Comment:
Is `node_modules` excluded automatically?

Reply:
AFAIK yes! I didn't see any `node_modules` in the logs when running `prettier --write`